src/style-spec/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/mapbox-gl-style-spec",
   "description": "a specification for mapbox gl styles",
-  "version": "13.2.0-beta.1",
+  "version": "13.3.0",
```

Comment:
13.2.0

Reply:
Oops, got confused and forgot that I had already updated this in the release-frappe branch. Thanks for pointing it out.

Reply:
13.2.0 has already been published to npm